src/SourceBuild/content/repo-projects/package-source-build.proj

```diff
@@ -39,21 +39,20 @@
     <!-- Create a layout directory for the files that are to be included in the artifacts tarball. Since there are a large number of files, this will use symlinks
          instead of copying files to make this execute quickly. -->
     <PropertyGroup>
-      <_SourceBuiltLayoutDir>$(BaseIntermediateOutputPath)artifacts-layout/</_SourceBuiltLayoutDir>
+      <_SourceBuiltLayoutDir>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', 'artifacts-layout'))</_SourceBuiltLayoutDir>
       <SourceBuildReferencePackagesDestinationDirName>SourceBuildReferencePackages</SourceBuildReferencePackagesDestinationDirName>
     </PropertyGroup>
-    <MakeDir Directories="$(_SourceBuiltLayoutDir)$(SourceBuildReferencePackagesDestinationDirName)" />
     <Copy SourceFiles="@(_AllRepoFiles)"
           DestinationFolder="$(_SourceBuiltLayoutDir)"
           UseSymbolicLinksIfPossible="true" />
-    <Copy SourceFiles="$(BaseIntermediateOutputPath)$(SourceBuiltVersionFileName)"
+    <Copy SourceFiles="$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)', '$(SourceBuiltVersionFileName)'))"
           DestinationFolder="$(_SourceBuiltLayoutDir)"
           UseSymbolicLinksIfPossible="true" />
     <Copy SourceFiles="$(CurrentSourceBuiltPackageVersionPropsPath)"
-          DestinationFiles="$(_SourceBuiltLayoutDir)PackageVersions.props"
+          DestinationFiles="$([MSBuild]::NormalizeDirectory('$(_SourceBuiltLayoutDir)', 'PackageVersions.props'))"
           UseSymbolicLinksIfPossible="true" />
     <Copy SourceFiles="@(_ReferencePackageFiles)"
-          DestinationFolder="$(_SourceBuiltLayoutDir)$(SourceBuildReferencePackagesDestinationDirName)"
+          DestinationFolder="$([MSBuild]::NormalizeDirectory('$(_SourceBuiltLayoutDir)', '$(SourceBuildReferencePackagesDestinationDirName)'))"
```

Comment:
I would also revert these two changes. Using the NormalizeDirectory function here isn't necessary and just overkill in regards to msbuild evaluation perf. Using NormalizePath and NormalizeDirectory msbuild functions is useful for properties and items. Here in this case, the trailing slash doesn't add any benefit as the msbuild Copy task handles directories well with or without a trailing directory separator.